lib/cli.js

```diff
@@ -88,6 +88,11 @@ internals.traverse = function (paths, options) {
             global._labScriptRun = false;
             file = Path.resolve(file);
             var pkg = require(file);
+
+            if (!pkg) {
```

Comment:
Ya, if you require the file it will execute the self-executing function like that